astrology.lic

```diff
@@ -162,32 +160,32 @@ class Astrology
     return if !Flags['rtr-expire'].nil? && !Flags['rtr-expire']
 
     @equipment_manager.empty_hands
```

Comment:
If I'm holding my telescope in my hands when I start astrology script, equipment manager "stows" the item which goes to a default container instead of my telescope storage.

Maybe it can check if we're holding a telescope and put away what's in your hands accordingly?

Reply:
That is so good. Thank you!

Reply:
I'll move it to draft now, and will fix this tonight.